README.md

```diff
@@ -102,10 +88,7 @@ You can now connect to your local Airflow web console at
 `http://localhost:8000/`.
 
 All DAGs are paused by default for local instances and our staging instance of Airflow.
-In order to submit a DAG via the UI, you'll need to toggle the DAG from "Off" to "On",
-but be very careful to check what DAG runs are generated (Browse > DAG Runs), since it may start
-generating backfill runs based on the DAG's configured start date, which could get very expensive
-(set `schedule_interval=None` in your DAG definition to prevent these scheduled runs).
+In order to submit a DAG via the UI, you'll need to toggle the DAG from "Off" to "On".
```

Comment:
Why remove these caveats here?

Reply:
The jobs no longer backfill by default via #301. I'm pretty comfortable with the behavior after running the `main_summary` dag several times in the last two weeks.